Boardless/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.24f1
-m_EditorVersionWithRevision: 2020.3.24f1 (79c78de19888)
+m_EditorVersion: 2020.3.25f1
+m_EditorVersionWithRevision: 2020.3.25f1 (9b9180224418)
```

Comment:
Another note, it's probably not a good idea to change the project version to match your unity version, just open it with the other version or install 24f1. I think 24f1 is standard for the class, and what we're using.